### PR TITLE
Update de_DE.po

### PR DESCRIPTION
**Pull request by Tasm-Devil**

### CODE REVIEW

File: locale/de_DE.po [hunk 1]
```diff
@@ -103,11 +103,11 @@ msgstr "leerer heap"
 msgid "syntax error in JSON"
 msgstr "Syntaxfehler in JSON"
 
-#: extmod/modure.c:265
+#: extmod/modure.c:161
 msgid "Splitting with sub-captures"
-msgstr "Teilen mit unter-captures"
+msgstr "Splitting with sub-captures"
```
Comment (dhalbert) [inline locale/de_DE.po]: Reverted to English?

File: locale/de_DE.po [hunk 2]
```diff
@@ -1061,15 +1062,15 @@ msgstr "mehrere **x sind nicht gestattet"
 
 #: py/compile.c:2271
 msgid "LHS of keyword arg must be an id"
-msgstr ""
+msgstr "LHS of keyword arg must be an id"
```
Comment (dhalbert) [inline locale/de_DE.po]: English

File: locale/de_DE.po [hunk 3]
```diff
@@ -1061,15 +1062,15 @@ msgstr "mehrere **x sind nicht gestattet"
...
 
 #: py/compile.c:2287
 msgid "non-keyword arg after */**"
-msgstr ""
+msgstr "non-keyword arg after */**"
```
Comment (dhalbert) [inline locale/de_DE.po]: English

File: locale/de_DE.po [hunk 4]
```diff
@@ -1061,15 +1062,15 @@ msgstr "mehrere **x sind nicht gestattet"
...
 
 #: py/compile.c:2291
 msgid "non-keyword arg after keyword arg"
-msgstr ""
+msgstr "non-keyword arg after keyword arg"
```
Comment (dhalbert) [inline locale/de_DE.po]: English

**Comment by tannewt** [comment]
I'm going to fix this up so I can merge it with my changes.

File: locale/de_DE.po (hunk 1 above)
Comment (Tasm-Devil) [reply locale/de_DE.po]: Only when its unclear for me. Better englich than bad translation.

**Pull request merged**